docs/source/getting-started/institutional-faq.md

```diff
@@ -0,0 +1,246 @@
+stack, and can serve a variety of computing enviroments, user interfaces, and
+computational hardware.
+
+Finally, JupyterHub can be deployed anywhere - on enterprise cloud infrastructure, on
+High-Performance-Computing machines, on local hardware, or even on a single laptop.
+
+# For IT
+
+## How would I set up JupyterHub on institutional hardware?
+
+That depends on what kind of hardware you've got. JupyterHub is flexible enough to be deployed
+on a variety of hardware, including in-room hardware, on-prem clusters, cloud infrastructure,
+etc.
+
+The most common way to set up a JupyterHub us to use a JupyterHub distribution, these are pre-configured
+and opinionated ways to set up a JupyterHub on particular kinds of infrastructure. The two distributions
+that we currently suggest are:
+
+* [Zero to JupyterHub for Kubernetes](https://z2jh.jupyter.org) is a scalable JupyterHub deployment and
+  guide that runs on Kubernetes. Better for larger or dynamic user groups (50-10,000) or more complex
+  compute/data needs.
+* [The Littlest JupyterHub](https://tljh.jupyter.org) is a lightweight JupyterHub that runs on a single
+  VM in the cloud. Better for smaller usergroups (4-80) or more lightweight computational resources.
+
+
+## Does JupyterHub run well in the cloud?
+
+Yes - most deployments of JupyterHub are run via cloud infrastructure and on a variety of cloud providers.
+Depending on the distribution of JupyterHub that you'd like to use, you can also connect your JupyterHub
+deployment with a number of other cloud-native services so that users have access to other resources from
+their interactive computing sessions.
+
+For example, if you use the [Zero to JupyterHub for Kubernetes](https://z2jh.jupyter.org) distribution,
+you'll be able to utilize container-based workflows of other technologies such as the [dask-kubernetes](https://kubernetes.dask.org/en/latest/)
+project for distributed computing.
+
+The Z2JH Helm Chart also has some functionality built in for auto-scaling your cluster up and down
+as more resources are needed - allowing you to utilize the benefits of a flexible cloud-based deployment.
+
+## Is JupyterHub secure?
+
+The short answer: yes. JupyterHub as a standalone application has been battle-tested at an institutional
+level for several years, and makes a number of "default" security decisions that are reasonable for most
+users.
+
+The longer answer: it depends on your deployment. Because JupyterHub is very flexible, it can be used
+in a variety of deployment setups. This often entails connecting your JupyterHub to **other** infrastructure
+(such as a [Dask Gateway service](https://gateway.dask.org/)). There are many security decisions to be made
+in these cases, and the security of your JupyterHub deployment will often depend on these decisions.
+
+If you are worried about security, don't hesitate to reach out to the JupyterHub community in the
```

Comment:
This will be a very helpful FAQ.  Conceding up front that this FAQ is not a security-focused document, one bit of feedback I have from an "institution" is that the Project needs to do more than say "ask on the forum" when it comes to securing Jupyter infrastructure.  There needs to be a similar FAQ or set of documented best practices in one place (not spread across all the projects) that you could refer to.  How can we help make that happen?

@rpwagner @tgmachina

Reply:
@rcthomas do you know of a document that we can refer to that covers security in JupyterHub deployments? I only said "ask on the forum" because I don't know of any-such document. I agree it should exist but am not qualified to write that content :-)

Reply:
We adapted @tgmachina’s work into a slide deck that we build on. I’ve been waiting to share it after a post a post about the training it was used for but I can share it now.

Reply:
For reference, here's the [slides](https://docs.google.com/presentation/d/178agvhzTrA14dP0GrN9Aa37EZIzc0kFb7fDbF_wpnPg). I could use some suggestions as to where a document that covers the material with more narrative text should live.

Reply:
@rpwagner if you have the content to write a post along the lines of "best practices in securing jupyterhub for institutions" I'd love to read it. Do you have any institutional blogs to post to? Or maybe the Jupyter Blog? Or potentially some contributions to the docs?

Reply:
(also slides need permissions changed)

Reply:
Slides permissions are updated.

The blog post was about the training we did, including @Carreau. I'm willing to draft what you describe, based on applying the security features we used. I can write it up somewhere in Markdown and share it.

Reply:
I really like those slides - lots of useful and relevant information in there, I'd love to see that as a blog post or white paper or something

Reply:
Wherever most appropriate, I'd like to see implementation details worked into the actual Jupyter documentation (in addition to a blog post and/or white paper). It can be a bit frustrating jumping between several sources trying to work on something when there's no centralized technical posting.

Maybe somewhere like this page: https://jupyter.readthedocs.io/en/latest/projects/content-projects.html# ?

Reply:
I'm all for getting it into the Jupyter documentation and since security cuts across multiple components in the community, having it part of the core Jupyter docs is better than it being just in JupyterHub or the Notebook Server docs.

Reply:
I'd also love to see this in the jupyter documentation wherever it fits